core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -316,8 +316,9 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
         for {
           costBreakdown <- executionServiceCluster.getCost(workflowRec, petUser)
           updatedWorkflowRec <-
-            // TODO CORE-217: if workflow is already in Aborted or Aborting, don't try to re-abort it
-            if (costBreakdown.cost > costCap) {
+            if (
+              costBreakdown.cost > costCap && costBreakdown.status != SubmissionStatuses.Aborted.toString && costBreakdown.status != SubmissionStatuses.Aborting.toString
```

Comment:
2 things
1. `costBreakdown.status` is a workflow status, not a submissions status
2. Why is `costBreakdown.status` stringly typed? Why isn't it using the enum?

Reply:
1. Fixed!  
2. I don't know!